packages/suite-desktop-core/scripts/sign-windows.ts

```diff
@@ -0,0 +1,18 @@
+exports.default = async function (configuration) {
```

Comment:
Shouldn't be async? See Files changed for other linter warnings as well.

Reply:
Yeah was just fixing it :) I took the script from electron documentation i guess its not really good one.. :D